docs/planning/netstandard-vnext/README.md

```diff
@@ -0,0 +1,117 @@
+  related improvements in .NET Core 2.1 as it allows managing buffers in a more
+  efficient way as it helps in reducing allocations and copying. `Span<T>` is
+  considered a primitive type and requires runtime and compiler support in order
+  to be fully leveraged. Please note that when refer to `Span<T>` in this
+  document, we really mean the family of span-related features which includes
+  other types such as `ReadOnlySpan<T>`, `Memory<T>`, `ReadOnlyMemory<T>`, and
+  the new `System.Buffers` namespace.
+
+* **Include core-APIs working with spans**. While `Span<T>` is available as a
+  .NET Standard compatible NuGet package (`System.Memory`) adding this package
+  cannot provide extend the members of .NET Standard types that deal with spans.
+  For example, .NET Core 2.1 added many APIs that allow working with spans, such
+  as `Stream.Read(Span<Byte>)`. Part of the value proposition to add span to
+  .NET Standard is to add the companion APIs as those cannot be added using a
+  NuGet package.
+
+* **General Goodness**. Since .NET Core was open sourced, the community and us
+  added many small features across the base class libraries such as
+  `System.HashCode` for combining hash codes or new overloads on
+  `System.String`. There are about ~800 new members in .NET Core that are on
+  types that are already in .NET Standard 2.0.
+
+## Decisions
+
+* We should **include** Brotli as HttpClient depends on it
+* We should **exclude** `DataAnnotations` (OOB today)
+* We should **exclude** `DiagnosticSource` (OOB today)
+* We should **exclude** `System.Reflection.Metadata` (OOB today)
+* We should **exclude** `System.Runtime.InteropServices.WindowsRuntime` (OOB
+  today)
+* We should **exclude** `System.Runtime.Loader` (currently a .NET Core only
+  concept and not ready for standardization)
+* We should exlude `RegexCompilationInfo` as it throws on .NET Core & Xamarin
+  and only works on .NET Framework. Since there is no version of .NET Framework
+  implementing the new version of the standard, so... yeah.
+* We should **exclude** `System.Threading.Tasks.Dataflow` (OOB today)
+* We should **exclude** `IApplicationResourceStreamResolver` (was plumbing for
+  Silverlight convergence. It's obsoleted.)
+* We should **exclude** `System.Xml.Serialization.SchemaImporter` (doesn't seem
+  useful)
+
+## Open issues
+
+* **Version number**. In contrast to .NET Standard 2.0, which was a massive
+  update in terms of APIs and the .NET Framework compatibility mode, this update
+  seems relatively incremental. Also, the payload is comparable to .NET Core
+  2.1, hence .NET Standard 2.1 seems to make the most sense.
+    - .NET Core and .NET Standard aren't the same thing. People will expect that
+      the next version is `3.0` as that's what we called .NET Core. However, the
+      API set is closer to `2.1` and argubly makes more sense within the context
+      of the standard.
```

Comment:
What would the plans be for incorporating .NET Core 3 API enhancements? Would that be .NET Standard vNext++?

Reply:
> What would the plans be for incorporating .NET Core 3 API enhancements? 

We want to avoid defining the standard with APIs that are in-flux as this violates our inclusion principles.

> Would that be .NET Standard vNext++?

Correct

Reply:
Wouldn't it be ill-advised to remove a lot of existing functionality and call it an incremental version? While I'm all for the removal of things like data annotations out of standard, calling it an incremental change while containing application breaking features seems wrong to me

Reply:
Sorry, I see the confusion. I'll update the document.

Exclusion here means "excluded from the set of candidates". These were never part of .NET Standard.